sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-kernel/coreos-kernel-6.12.48.ebuild

```diff
@@ -89,6 +100,77 @@ src_compile() {
 
 	tc-export PKG_CONFIG
 	"${ESYSROOT}"/usr/bin/update-bootengine -k "${KV_FULL}" -o "${S}"/build/bootengine.cpio "${BE_ARGS[@]}" || die
+	# Copy full initrd over to /usr as filesystem image
+	mkdir "${S}"/build/bootengine || die
+	pushd "${S}"/build/bootengine || die
+	sudo lsinitrd --unpack "${S}"/build/bootengine.cpio || die
+	sudo mksquashfs . "${S}"/build/bootengine.img -noappend -xattrs-exclude ^btrfs. || die
+	popd || die
+	# Create minimal initrd
+	if use amd64; then
+		microcode=$(cpio -t < "${S}"/build/bootengine.cpio 2>&1 > /dev/null | cut -d " " -f 1)
```

Comment:
This looks scary. `lsinitrd` does have an `--unpackearly` option!

Reply:
We can try that out - I didn't find that much information, for example, I'm not sure if this `early_cpio` file is also required to be there or if this is not really required by the kernel.

Reply:
It's not used by the kernel. I think it's only used by Dracut's `lsinitrd`. Yes, I guess that's a little circular!

One thing to note. The main cpio can be compressed, but the early cpio must not be.

Reply:
I've done the change